src/lib.rs

```diff
@@ -28,6 +28,18 @@ pub struct Apa102<SPI> {
     spi: SPI,
     end_frame_length: u8,
     invert_end_frame: bool,
+    pixel_type: PixelType,
+}
+
+/// What order to transmit pixel colors. Different Dotstars
+/// need their pixel color data sent in different orders.
+pub enum PixelType {
```

Comment:
Just a suggestion, feel free to ignore it, but in my opinion `PixelOrder` fits better

Reply:
I was actually thinking the same thing. Good idea, will update!